### PR TITLE
Clarifying that if user downloaded from Tech Hub, it includes sbt dist

### PR DESCRIPTION
**Pull request by rstento**
Similar to the change for the Scala example.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 # Play Hello World Web Tutorial for Java
 
-To follow the steps in this tutorial, you will need the correct version of Java and a build tool. You can build Play projects with sbt or Gradle. Since sbt works particularly well with Play, we recommend trying this example with sbt. The template requires:
+To follow the steps in this tutorial, you will need the correct version of Java and a build tool. You can build Play projects with any Java build tool. Since sbt takes advantage of Play features such as auto-reload, the tutorial describes how to build the project with sbt. 
+
```
Comment (octonato) [inline README.md:4]: I don't know how we came to that situation, but previous text was mentioned:
> You can build Play projects with sbt or Gradle

That was not true becasue this project does not contain a gradle build file. Maybe it was removed at some point. 

Theoretically it's possible to build it with any Java build tool, but we need a build definition for each tool, which is not the case here. We can only build with sbt.

Comment (octonato) [reply README.md:4]: I just realised that this is a fairly new project. Only 18 commits. I don't think it has ever had a Gradle build definition. Was this README copied from somewhere?

Comment (octonato) [reply README.md:4]: @rstento, I sent a PR (https://github.com/playframework/play-java-hello-world-tutorial/pull/18) with the gradle build. 

We should add the sentence "You can build Play projects with sbt or Gradle" back after merging #18.

Comment (octonato) [reply README.md:4]: I will do the same for the scala variant

**Comment by rstento** [comment]
Thanks @Renato Cavalcanti <renato@lightbend.com>!
Ruth Stento
User Experience Lead
Lightbend

512-696-1448
<http://lightbend.com>

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
+Prerequisites include:
 
 * Java Software Developer's Kit (SE) 1.8 or higher
-* sbt 0.13.15 or higher (we recommend 1.2.3)
+* sbt 0.13.15 or higher (we recommend 1.2.3) Note: if you downloaded this project as a zip file from https://developer.lightbend.com, the file includes an sbt distribution for your convenience.
```
Comment (marcospereira) [inline README.md:8]:
```suggestion
* sbt 0.13.15 or higher (we recommend 1.2.7) Note: if you downloaded this project as a zip file from https://developer.lightbend.com, the file includes an sbt distribution for your convenience.
```

**Review by marcospereira — COMMENTED** [review]
sbt 1.2.7.

**Review by dwijnand — APPROVED** [review]
LGTM.

We can upgrade sbt versions and restate that specifically in this project you can use sbt or Gradle, in follow-up PRs.

**Pull request merged**